Sentinel/Classes/Core/Internal/DeviceTool.swift

```diff
@@ -52,11 +47,38 @@ extension DeviceTool {
 
 private extension DeviceTool {
 
+    static var section: CustomInfoTool.Section {
+        #if os(macOS)
+        CustomInfoTool.Section(title: "Device", items: [
+            .init(title: "Model", value: getModelIdentifier() ?? ""),
+            .init(title: "Name", value: ProcessInfo.processInfo.hostName),
+            .init(title: "System version", value: systemVersion),
+            .init(title: "UUID", value: hardwareDeviceUUID ?? "???")
+        ])
+        #else
+        CustomInfoTool.Section(title: "Device", items: [
+            .init(title: "Model", value: UIDevice.current.model),
+            .init(title: "Name", value: UIDevice.current.name),
+            .init(title: "System version", value: systemVersion),
+            .init(title: "UUID", value: UIDevice.current.identifierForVendor?.uuidString ?? "???"),
+            .init(title: "Battery state", value: batteryState),
+            .init(title: "Proximity state", value: UIDevice.current.proximityState ? "Close" : "Far")
```

Comment:
Replace `.init` methods with real class names.

Also, additionally, if the objects of the array are the same, you could instead return the array from a method instead creating the whole object inside the `if-else` branches. Even better approach here would be a factory method or abstract factory (due to other methods), but I believe this can be added and improved in other PRs.